MainWindow.cpp

```diff
@@ -85,6 +87,14 @@ void diagnosticHandler(QtMsgType type, const QMessageLogContext &context, const
   }
 }
 
+void MainWindow::nameMap(const char *name, TypeCase type)
+{
+   QVariant nameVar = tr(name);
+//   defaultGroupNames.emplace_back(nameVar, type);

```

Comment:
Don't leave commented code in the source, it just rots. If it's not needed, delete it.